src/test/FedoraServices.test.tsx

```diff
@@ -27,6 +27,7 @@ const goToDetailsStep = async () => {
   await clickNext(); // Kernel
   await clickNext(); // Firewall
   await clickNext(); // Services
+  await clickNext(); // First boot
```

Comment:
Assuming the First boot step should be available in the Fedora service as well. If it's an incorrect assumption let me know and I'll switch this for hiding the step for Fedora env instead.